news-app/src/components/SingleArticlesByTopic.jsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import * as API from '../API.js'
+import SingleArticleByTopicItem from "./SingleArticleByTopicItem.jsx"
+
+
+const SingleArticlesByTopic = ()=>{
```

Comment:
I'm finding the name SingleArticles a bit confusing in this context because this is describing a list of articles - is there a clearer name you could use?